### PR TITLE
Remove/fix broken queries

### PR DESCRIPTION
**Pull request by labkey-martyp**
#### Rationale
Remove/fix the last queries not passing validation. Turn on query validation after study import in auto tests.

#### Changes
* Remove unused queries
* Fix researchUltrasoundsWithReview.sql
* Turn on query validation

### CODE REVIEW

**Review by labkey-jeckels — APPROVED** [review]
Changes look good and TeamCity seems happy, but let's be sure to get WNPRC's signoff before merging.

**Review by dnicolalde — COMMENTED** [review]
looks good.

**Pull request merged**
